ansible/roles/satellite_slave/templates/satellite-slave-config.clj.j2

```diff
@@ -12,11 +12,19 @@
 ;; See the License for the specific language governing permissions and
 ;; limitations under the License.
 
-(def mesos-work-dir "/tmp/mesos")
+(def custom-comets [])
+
+(defn build-comet
```

Comment:
Yeah I had this in the .jar originally and ended up moving it out into this external file (since it relates only to the mechanism of loading the settings that are then consumed by satellite proper).  I don't feel strongly about it living here though, and can definitely move it into the jar if you prefer.